fhir-persistence-blob-app/src/main/java/com/ibm/fhir/persistence/blob/app/Main.java

```diff
@@ -0,0 +1,229 @@
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
+
+    // properties for connecting to the database
+    private final Properties databaseProperties = new Properties();
+
+    private String fhirConfigDir;
+    private String tenantId;
+    private String dsId;
+    private DbType dbType;
+    private boolean reconcile;
+    private boolean createContainer;
+    private boolean dryRun = true;
+    private String continuationToken = null;
+
+    // A list of resources for which we want to read the blob contents
+    private final List<String> resourceList = new ArrayList<>();
+
+    // Stop scanning after this number of seconds. Use continuationToken to continue
+    private int maxScanSeconds = 120;
+
+    /**
+     * Parse the command line arguments
+     * @param args
+     */
+    protected void parseArgs(String[] args) {
+        for (int i=0; i<args.length; i++) {
+            String arg = args[i];
+            switch (arg) {
+            case "--fhir-config-dir":
+                if (++i < args.length) {
+                    this.fhirConfigDir = args[i];
+                } else {
+                    throw new IllegalArgumentException("Missing value for --fhir-config-dir");
+                }
+                break;
+            case "--continuation-token":
+                if (++i < args.length) {
+                    this.continuationToken = args[i];
+                } else {
+                    throw new IllegalArgumentException("Missing value for --continuation-token");
+                }
+                break;
+            case "--db-properties":
+                if (++i < args.length) {
+                    readDatabaseProperties(args[i]);
+                } else {
+                    throw new IllegalArgumentException("Missing value for --db-properties");
+                }
+                break;
+            case "--db-type":
+                if (++i < args.length) {
+                    this.dbType = DbType.from(args[i]);
+                } else {
+                    throw new IllegalArgumentException("Missing value for --db-type");
+                }
+                break;
+            case "--tenant-id":
+                if (++i < args.length) {
+                    this.tenantId = args[i];
+                } else {
+                    throw new IllegalArgumentException("Missing value for --tenant-id");
+                }
+                break;
+            case "--ds-id":
+                if (++i < args.length) {
+                    this.dsId = args[i];
+                } else {
+                    throw new IllegalArgumentException("Missing value for --ds-id");
+                }
+                break;
+            case "--reconcile":
+                this.reconcile = true;
+                break;
+            case "--create-container":
+                this.createContainer = true;
+                break;
+            case "--confirm":
+                this.dryRun = false;
```

Comment:
Since `--confirm` and `--dry-run` both affect the same value, it would be good to explain what happens if both are specified (especially if order matters)

Reply:
I'll add to the README. I added `--confirm` as an extra requirement because I wanted the `--dry-run` to be the default (because reconciliation can delete stuff).

Reply:
Perhaps I'll rename `--confirm` to `--no-dry-run` which might make it a little easier. Last one wins.

Reply:
Changed to --no-dry-run and updated the README to describe the behavior.